Theory/ELF.md

```diff
@@ -1,32 +1,32 @@
 Executable and Linkable Format
 ================================================================================
 
-ELF (Executable and Linkable Format) is a standard file format for executable files, object code, shared libraries, and core dumps. Linux, as well as, many other UNIX-like operating systems uses this format. Let's look on the structure of ELF-64 File Format and some defintions in the linux kernel source code related with it.
+ELF (Executable and Linkable Format) is a standard file format for executable files and shared libraries. Linux and many UNIX-like operating systems uses this format. Let's look at the structure of the ELF-64 Object File Format and some definitions in the linux kernel source code which related with it.
 
```

Comment:
I don't agree with removing the "object code" and "core dump"! Because those two file types are of ELF format.
My major language is Chinese, not English, but I think "ELF Object file" is not appropriate. 
"which" is not right , you can change it with "which are" 
What do you think ?